.travis.yml

```diff
@@ -25,39 +25,76 @@ env:
 before_install: curl -Ls https://git.io/jabba | bash && . ~/.jabba/jabba.sh
 install: jabba install "$TRAVIS_JDK" && jabba use "$_" && java -Xmx32m -version
 
+stages:
+  - name: prepare
+    if: NOT tag =~ ^v
+  - name: test
```

Comment:
Splitting this in 'prepare' and 'test' makes travis a bit slower: it means the 12-minute 'sbt scripted' job will not start before the 5-minute 'prepare' phase is done.

I'm OK with that though, we can always optimize further later.

Reply:
I will put `scripted` to preparte

Reply:
Hmm, it seems not that good idea now, sinc tests will wait for scripted job.
Is there a chance to split `scripted` job into several smaller jobs ?

Reply:
Another option for `scripted` job will be to use pre compiled classes saved from previous stage
does `scripted` use a single scala and JDK version ( according to logs - i suppose, yes)